packages/x-components/src/components/__tests__/location-injector.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { mount, Wrapper, VueClass } from '@vue/test-utils';
+import Vue from 'vue';
+import { Component, Inject } from 'vue-property-decorator';
+import { getDataTestSelector } from '../../__tests__/utils';
+import { FeatureLocation } from '../../types';
+import LocationInjector from '../location-injector.vue';
+
+@Component({
+  template: `<div data-test="inject"/>`
+})
+class Child extends Vue {
+  @Inject('location')
+  public location!: FeatureLocation;
+}
+
+/**
+ * Renders the `LocationInjector` component, exposing a basic API for testing.
```

Comment:
Whenever it is possible, try to link documentation parts like the name of the components so it is easier to refactor if necessary:

```suggestion
 * Renders the {@link LocationInjector} component, exposing a basic API for testing.
```